src/uv-common.c

```diff
@@ -155,6 +155,82 @@ static const char* uv__unknown_err_code(int err) {
   return copy != NULL ? copy : "Unknown system error";
 }
 
+int uv_translate_posix_error(int sys_errno) {
+  switch (sys_errno) {
```

Comment:
If there is an errno we forgot to map, then we need to map it :-) As for the implementation, even if we go with a single function, it will have 2 implementations, the Unix side, which returns the negated errno, and the Windows one, which translates win32 errors into uv errors.